docs/dev/android/setup-android-build-environment.md

```diff
@@ -1,89 +1,75 @@
-# Setup the Android Build Environment
+<!-- MarkdownTOC autolink="true" -->
```

Comment:
Is this auto-generated by your local tool? 
It will [be rendered into a webpage](https://badboy.github.io/glean.rs/book/dev/android/setup-android-build-environment.html) and therefore the heading should be first. 
I'm also not sure we need an extensive inline-ToC for a now rather short doc.

Reply:
Oh, ok, removing this.